servers/zms/src/main/java/com/yahoo/athenz/zms/store/jdbc/JDBCConnection.java

```diff
@@ -870,7 +870,33 @@ public boolean deleteDomainTemplate(String domainName, String templateName, Stri
         Collections.sort(templates);
         return templates;
     }
-    
+
+    @Override
+    public Map<String, List<String>> getDomainFromTemplateName(Map<String, Integer> templateNameAndLatestVersion) {
+        final String caller = "getDomainsFromTemplate";
+        Map<String, List<String>> domainNameTemplateListMap = new HashMap<>();
+
+        try (PreparedStatement ps = con.prepareStatement(ZMSUtils.generateDomainTemplateVersionQuery(templateNameAndLatestVersion))) {
+            try (ResultSet rs = executeQuery(ps, caller)) {
+                while (rs.next()) {
+                    if (domainNameTemplateListMap.get(rs.getString(ZMSConsts.DB_COLUMN_NAME)) != null) {
```

Comment:
instead of calling rs.getString multiple times in the if block we should just get those two values and assign to local variables and then do the checks with those values which makes the code more readable.

Reply:
Done.